appengine/cloudsql-postgres/README.md

```diff
@@ -0,0 +1,85 @@
+# Ruby Cloud SQL PostgreSQL sample on Google App Engine flexible environment
+
+This sample demonstrates how to use
+[Google Cloud SQL for PostgreSQL][postgres]
+(or any other SQL server) on [Google App Engine flexible environment][flexible].
+
+## Setup
+
+Before you can run or deploy the sample, you will need to do the following:
+
+ 1. [Create a Cloud SQL for PostgreSQL instance](https://cloud.google.com/sql/docs/postgres/create-instance).
+ 
+ 1. Create a new database in the instance databases page in the Google Cloud Platform Console.
```

Comment:
Documentation to link to for creating/managing databases using PostgreSQL. https://cloud.google.com/sql/docs/postgres/create-manage-databases